BeerProject/src/main/java/wawer/kamil/beerproject/service/BreweryService.java

```diff
@@ -0,0 +1,32 @@
+package wawer.kamil.beerproject.service;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
+import wawer.kamil.beerproject.model.Brewery;
+import wawer.kamil.beerproject.exceptions.InvalidImageParameters;
+import wawer.kamil.beerproject.exceptions.NoContentException;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface BreweryService {
```

Comment:
Jeżeli twoje wyjątki będą rozszerzeniem klasy `RuntimeException` a nie `Exception` to nie będziesz musiał ich rzucać wyżej a i tak będą obsługiwane w Advicerze - kod będzie zdecydowanie czytelniejszy